src/env.ts

```diff
@@ -531,6 +531,11 @@ const envDefinitions = {
     parser: intParser,
     desc: "version used for the platform api",
   },
+  MARKET_API_URL: {
+    def: "https://api.coingecko.com/api/v3", // @TODO replace with prod API proxy whenever ready
```

Comment:
that TODO makes this non mergeable, correct?

Reply:
yes until we get actual infra proxy :grimacing: